packages/client/src/styles/globals.css

```
@@ -3845,6 +3851,66 @@ strong {
+    transform: translateY(8px) scale(0.96);
+  }
+  to {
+    opacity: 1;
+    transform: translateY(0) scale(1);
+  }
+}
+
+.mari-chat-drop-zone {
+  --mari-chat-drop-accent: var(--mari-panel-gradient-end, var(--primary));
+  --mari-chat-drop-accent-soft: var(--mari-panel-gradient-start, var(--primary));
+  border: 0.25rem dotted color-mix(in srgb, var(--mari-chat-drop-accent) 80%, transparent);
```

Stylelint (17.14.1)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)